com.unity.render-pipelines.high-definition/Documentation~/whats-new-10-0.md

```diff
@@ -205,6 +205,14 @@ HDRP, being a high-end modern renderer, contains a lot of compute shader passes.
 
 Planar reflection probe filtering is a process that combines the result of planar reflection and surfaces smoothness. Up until this version, the implementation for planar reflection probe filtering did not always produce results of fantastic quality. This version of HDRP includes a new implementation that is closer to being physically-based and improves on the image quality significantly.
 
+### Screen space reflection
+
+[Screen Space Reflection](Override-Screen-Space-Reflection.md) effect always use the color pyramid generate after the Before Refraction transparent pass. Thus the color buffer only includes transparent GameObjects that use the **BeforeRefraction** [Rendering Pass](Surface-Type.md).
+
+### Distortion
+
+Distortion effect now support a rough distortion effect which is disabled by default. Disabling Rough Distortion allow to save the cost of generating a color pyramid for the effect and instead rely on a copy of the screen.
```

Comment:
The distortion effect now supports a rough distortion which is disabled by default. Disabling Rough Distortion saves resources as the effect does not generate a color pyramid and instead uses a copy of the screen.